packages/camera/camera_web/example/README.md

```diff
@@ -0,0 +1,21 @@
+# Testing
+
+This package utilizes the `integration_test` package to run its tests in a web browser.
+
+See [flutter.dev > Integration testing](https://flutter.dev/docs/testing/integration-tests) for more info.
+
+## Running the tests
+
+Make sure you have updated to the latest Flutter master.
```

Comment:
Instead of repeating these instructions here, see the latest README I've written, pointing people to the official docs: https://github.com/flutter/plugins/commit/731b93d52e792e782e47d3bdcd237cd2138e56ad#diff-cbed0883b6c7982649a43b887188ce49654e62bfd6787eef04842c54565a02f8